console/src/main/scripts/plugins/metrics/html/directives/alert-summary.html

```diff
@@ -23,7 +23,7 @@
     <strong>Non Heap Memory</strong>: The Non Heap memory usage was {{alert.avg / 1024 / 1024 | number:2}} MB.
   </div>
   <div ng-switch-when="GARBA">
-    <strong>Garbage collection</strong>: The GC averaged more than {{alert.avg | number:2}} ms per minute.
+    <strong>Garbage collection</strong>: The GC averaged more than {{alert.avg | number:0}} ms per minute.
```

Comment:
Is it possible to change the message to adapt to the size of the alert.avg? For instance, if it's higher than 1000ms, then report in seconds with up to 2 decimal places. If it's higher than 60s, report it in minutes, and so on.

I'd prefer to read "2.3s" as opposed to "2,300.00ms".